src/test/java/teammates/test/cases/ui/pagedata/InstructorCourseDetailsPageDataTest.java

```diff
@@ -0,0 +1,102 @@
+        assertFalse(pageData.getCourseRemindButton().getAttributes().isEmpty());
+        assertNull(pageData.getCourseRemindButton().getContent());
+        assertNotNull(pageData.getGiveCommentButton());
+        assertFalse(pageData.getGiveCommentButton().getAttributes().isEmpty());
+        assertNotNull(pageData.getGiveCommentButton().getContent());
+        assertNotNull(pageData.getCourseDetails());
+        assertNotNull(pageData.getCurrentInstructor());
+        
+        ______TS("test current instructor doesn't have any permission for the course");
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_COMMENT_IN_SECTIONS, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COMMENT_IN_SECTIONS, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS, false);
+        curInstructor.privileges.updatePrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS, false);
+        pageData.init(curInstructor, courseDetails, instructors, students);
+
+        assertEquals(instructors.size(), pageData.getInstructors().size());
+        assertEquals(students.size(), pageData.getStudentsTable().getRows().size());
+        assertEquals(4, pageData.getStudentsTable().getRows().get(0).getActions().size());
+        assertEquals(2, pageData.getStudentsTable().getRows().get(0).getCommentActions().size());
+        assertNotNull(pageData.getCourseRemindButton());
+        assertFalse(pageData.getCourseRemindButton().getAttributes().isEmpty());
+        assertNull(pageData.getCourseRemindButton().getContent());
+        assertNotNull(pageData.getGiveCommentButton());
+        assertFalse(pageData.getGiveCommentButton().getAttributes().isEmpty());
+        assertNotNull(pageData.getGiveCommentButton().getContent());
+        assertNotNull(pageData.getCourseDetails());
+        assertNotNull(pageData.getCurrentInstructor());
```

Comment:
the difference is the buttons are disabled because the instructor doesn't have permission to click on those buttons. I'll add test for "disabled" attribute.